examples/react-app/package.json

```diff
@@ -28,7 +28,6 @@
   "scripts": {
     "start": "GENERATE_SOURCEMAP=false react-app-rewired start",
     "build": "react-app-rewired build",
-    "test": "react-app-rewired test",
```

Comment:
should this be removed?

Reply:
Yes, it's on purpose. There aren't any tests for the example package.

`turbo run test` will run `test` on every npm project under `/examples` or `/packages`. This was an easy way to avoid hard coding a filter at the root level.